generator/resources/eks_daemon_test_matrix.json

```diff
@@ -2,7 +2,7 @@
   {
     "k8s_version": "1.24",
     "ami": "AL2_x86_64",
-    "instanceType":"t3a.medium",
+    "instanceType":"t3.medium",
```

Comment:
Any reason for the change from t3a to t3?

Reply:
This is because it was already using t3.medium. See the main.tf file.